snyk-monitor/values.yaml

```diff
@@ -8,18 +8,25 @@ monitorSecrets: snyk-monitor
 certsConfigMap: snyk-monitor-certs
 registriesConfConfigMap: snyk-monitor-registries-conf
 
-# The ConfigMap to use for loading policies into snyk-monitor.
-workloadPoliciesMap: snyk-monitor-workload-policies
+# An external ConfigMap to use for loading policies into snyk-monitor.
+# When using this the default policy shouldn't be deployed, set `defaultWorkloadPolicy.enabled: false`.
```

Comment:
Could you please update this doc line? 
I would suggest to write the following - "If not set, defaultWorkloadPoliciesMap will be used".

Reply:
sure, no problem

Reply:
done